node/src/components/consensus/protocols/highway/synchronizer.rs

```diff
@@ -210,12 +215,15 @@ impl<I: NodeIdT, C: Context + 'static> Synchronizer<I, C> {
 
     // Returns number of elements in `vertex_deps` queue.
     fn vertex_deps_len(&self) -> u64 {
-        self.vertex_deps.iter().map(|(_, pv)| pv.len()).sum()
+        self.vertices_awaiting_deps
+            .iter()
+            .map(|(_, pv)| pv.len())
+            .sum()
     }
 
     // Returns number of elements in `vertices_to_be_added` queue.
     fn vertices_to_be_added_len(&self) -> u64 {
```

Comment:
```suggestion
    fn vertices_no_deps_len(&self) -> u64 {
```

Reply:
Changed in [`d3e8a82` (#1312)](https://github.com/CasperLabs/casper-node/pull/1312/commits/d3e8a82bae9fc19cf5a7839a640b0a219720e6c5)